lnbits/extensions/boltz/tasks.py

```diff
@@ -0,0 +1,154 @@
+import asyncio
+
+import httpx
+from loguru import logger
+
+from lnbits.core.models import Payment
+from lnbits.core.services import check_transaction_status
+from lnbits.tasks import register_invoice_listener
+
+from .boltz import (
+    create_claim_tx,
+    create_refund_tx,
+    get_swap_status,
+    start_confirmation_listener,
+    start_onchain_listener,
+)
+from .crud import (
+    get_all_pending_reverse_submarine_swaps,
+    get_all_pending_submarine_swaps,
+    get_reverse_submarine_swap,
+    get_submarine_swap,
+    update_swap_status,
+)
+
+"""
+testcases for boltz startup
+A. normal swaps
+  1. test: create -> kill -> start -> startup invoice listeners -> pay onchain funds -> should complete
+  2. test: create -> kill -> pay onchain funds -> start -> startup check  -> should complete
+  3. test: create -> kill -> mine blocks and hit timeout -> start -> should go timeout/failed
+  4. test: create -> kill -> pay to less onchain funds -> mine blocks hit timeout -> start lnbits -> should be refunded
+
+B. reverse swaps
+  1. test: create instant -> kill -> boltz does lockup -> not confirmed -> start lnbits -> should claim/complete
+  2. test: create instant -> kill -> no lockup -> start lnbits -> should start onchain listener -> boltz does lockup -> should claim/complete (difficult to test)
+  3. test: create -> kill -> boltz does lockup -> not confirmed -> start lnbits -> should start tx listener -> after confirmation -> should claim/complete
+  4. test: create -> kill -> boltz does lockup -> confirmed -> start lnbits -> should claim/complete
+  5. test: create -> kill -> boltz does lockup -> hit timeout -> boltz refunds -> start -> should timeout
+"""
```

Comment:
Why are these here? Shouldn't it be in `/tests/` rather?

Reply:
currently i have no good way to test the startup function, so i put i there for me to remember for manual testing